app/controllers/concerns/hyrax/local_file_downloads_controller_behavior.rb

```diff
@@ -65,7 +66,7 @@ def prepare_local_file_headers
         response.headers['Content-Type'] = local_file_mime_type
         response.headers['Content-Length'] ||= local_file_size.to_s
         # Prevent Rack::ETag from calculating a digest over body
-        response.headers['Last-Modified'] = local_file_last_modified.utc.strftime("%a, %d %b %Y %T GMT")
+        response.headers['Last-Modified'] ||= local_file_last_modified.httpdate
```

Comment:
This should probably be tested.
I'm also unsure about the `||=`.  What would be previously setting `Last-Modified` and should it really be preferred over `local_file_last_modified.httpdate`?  The comment above leads me to believe that it shouldn't but maybe the comment just needs to be updated.

Reply:
The call to `stale?` will add a Last-Modified header before this code executes, but I think you're right that the conditional assignment isn't really needed.